src/entities/trades/0x/0xTrade.ts

```diff
@@ -179,11 +109,13 @@ export class ZeroXTrade extends TradeWithSwapTransaction {
         ? currencyAmountIn.currency.symbol
         : tokenIn.address
 
+      // slippagePercentage for the 0X API needs to be a value between 0 and 1, others have between 0 and 100
       const response = await fetch(
-        `https://api.0x.org/swap/v1/quote?buyToken=${buyToken}&sellToken=${sellToken}&sellAmount=${
+        `${apiUrl}swap/v1/quote?buyToken=${buyToken}&sellToken=${sellToken}&sellAmount=${
           amountIn.raw
-        }&slippagePercentage=${maximumSlippage.toFixed(3)}`
+        }&slippagePercentage=${new Percent(maximumSlippage.numerator, '10000').toFixed(2)}`
```

Comment:
Instead of '10000' I would do something like maximumSlippage.denominator * 100.
We shouldn't assume that coincides with 100, right?

Reply:
yes, I agree

Reply:
@jorgedclopes changed